docs/examples/search_vector_similarity_examples.ipynb

```diff
@@ -19,39 +23,126 @@
        "b'OK'"
       ]
      },
-     "execution_count": 1,
+     "execution_count": 2,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
     "import redis\n",
-    "from redis.commands.search.field import VectorField\n",
+    "\n",
+    "from redis.commands.search.indexDefinition import (\n",
+    "    IndexDefinition,\n",
+    "    IndexType\n",
+    ")\n",
     "from redis.commands.search.query import Query\n",
+    "from redis.commands.search.field import (\n",
+    "    TagField,\n",
+    "    VectorField\n",
+    ")\n",
+    "\n",
+    "r = redis.Redis(host=\"localhost\", port=6379)\n",
+    "\n",
+    "INDEX_NAME = \"index\"                       # Vector Index Name\n",
+    "VECTOR_DIMENSIONS = 1536                   # Number of Vector Dimensions\n",
+    "DOC_PREFIX = \"doc:\"                        # RediSearch Key Prefix for the Index\n",
     "\n",
-    "r = redis.Redis(host='localhost', port=36379)\n",
+    "# Schema\n",
+    "schema = (\n",
+    "    TagField(\"tag\"),\n",
+    "    VectorField(\"vector\",                  # Vector Field Name\n",
+    "        \"FLAT\", {                          # Vector Index Type: FLAT or HNSW\n",
+    "            \"TYPE\": \"FLOAT32\",             # FLOAT32 or FLOAT64\n",
+    "            \"DIM\": VECTOR_DIMENSIONS,      # Number of Vector Dimensions\n",
+    "            \"DISTANCE_METRIC\": \"COSINE\",   # Vector Search Distance Metric\n",
+    "            \"INITIAL_CAP\": 1000            # Helps with memory planning and allocation\n",
+    "        }\n",
+    "    ),\n",
+    ")\n",
+    "# NOTE: sometimes users experience issues with the INITIAL_CAP param.\n",
```

Comment:
@tylerhutcherson you can drop this, we have fixed it at https://github.com/RediSearch/RediSearch/pull/3397